src/offsets.jl

```diff
@@ -176,27 +346,18 @@ function tai_offset(::BarycentricDynamicalTime, ep, elong, u, v)
 
     # TDB-TT in seconds.
     w = wt + wf + wj
-
-    tai_offset(TT, ep) + w
+    return -w
```

Comment:
Just do
```julia
return -(wt + wf + wj)
```
?  Or you prefer to show the intermediate step?

Reply:
During development it is nice to have the lines separated so one can quickly prefix `@show` for debugging. But for the final result I do not really care.